### DISCUSSION TITLE
Autocomplete for method overwrites could add missing imports

### DISCUSSION THREAD

**Discussion by cdce8p** [microsoft/pylance-release#2590] [Enhancement]
## Environment data

-   Language Server version: 2021.1.2
-   OS and version: darwin x64
-   Python version: 3.9.1 
-   python.analysis.indexing: true
-   python.analysis.typeCheckingMode: off

## Expected behaviour

It would be helpful if pylance could automatically add missing imports for parameter type hints when using autocomplete for an overwritten function. In some cases it can be quite tedious to go back and add them manually.

## Actual behaviour

Only the function parameter and their type hints are added.

## Logs
--

## Code Snippet / Additional information

```py
from argparse import Action

class MyAction(Action):
    def __call__  # autocomplete here

## Result
class MyAction(Action):
    def __call__(self, parser: ArgumentParser, namespace: Namespace, values: Union[Text, Sequence[Any], None], option_string: Optional[Text]) -> None:
        return super().__call__(parser, namespace, values, option_string=option_string)

# Missing imports
from argparse import ArgumentParser, Namespace
from typing import Union, Sequence, Any, Optional, Text
```

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
